book/index.qmd

```diff
@@ -141,7 +141,8 @@ We will begin by looking at advanced technical details in @sec-technical that ar
 @sec-interpretation will discuss different packages that are compatible with `mlr3` to provide model-agnostic interpretability for feature importance and local explainability of individual predictions.
 @sec-special will then delve into detail on domain-specific methods that are implemented in our extension packages including survival analysis, density estimation, spatio-temporal analysis, and more.
 Readers may choose to selectively read sections in this chapter depending on your use case (i.e., if you have domain-specific problems to tackle), or to use these as introductions to new domains to explore.
-Finally, @sec-fairness will introduce algorithmic fairness, which includes specialized measures and methods to identify and reduce algorithmic biases.
+Then, @sec-fairness will introduce algorithmic fairness, which includes specialized measures and methods to identify and reduce algorithmic biases.
+Finally, @sec-predict-sets will explain how to evaluate algorithms on different predict sets and how to activate validation and early stopping for `mlr3` learners.
```

Comment:
Could you add a few words here on what a predict set is please? In particular to contrast it with the earlier chapters on evaluation.